mantle/cmd/kola/kola.go

```diff
@@ -636,20 +636,29 @@ func getParentFcosBuildBase(stream string) (string, error) {
 		if err != nil {
 			return "", err
 		}
-
+		//as we build for multi-architectures now and inorder to allow failures to be non-fatal, enable them to promote to non-prod streams when any specific previous release is unavailable for that arch.
+		//in that case, releases are searched in reverse order and the most recent release which has the secondary arch is considered.
```

Comment:
Let's add a space after the `//` here in these comments. Also can you wrap them at 70 or 80 characters so there isn't such long lines?